lib/views/password_generation.dart

```diff
@@ -75,11 +80,16 @@ class _PasswordGenerationPageState extends State<PasswordGenerationPage> {
   }
 
   Future<void> encryptMnemonic(BuildContext context) async {
-    await MnemonicEncryptor.encryptMnemonic(widget.mnemonic!, passwordController.text);
-    final api = WalletApi();
-    api.importWallet(
+    await MnemonicEncryptor.encryptMnemonic(
+        widget.mnemonic!, passwordController.text);
```

Comment:
I think you have the line length set to 80 chars instead of 120

Reply:
✅